cloudfront/cdn_code/variables.tf

```diff
@@ -0,0 +1,268 @@
+variable "cloudfront_distributions" {
+  type = map(object({
+    #    aliases             = optional(list(string))
```

Comment:
why aliases is commented on module

Reply:
that was done by-mistake sir, please check now.